### ISSUE TITLE
flutter pub run build_runner build: 0 file output

### ISSUE THREAD

**Issue by seyaobey** [comigor/artemis#281]
**Before reporting a bug, please test the beta branch!**

## Bug description
Running the command `flutter pub run build_runner build` produces no file. This is the console output:
```
[INFO] Succeeded after 88ms with 0 outputs (0 actions)
```

## Specs
Artemis version: `artemis: '>=6.0.0 <7.0.0'`
<details>
<summary>build.yaml:</summary>
  
```yaml
# Please paste your `build.yaml` file
targets:
  $default:
    sources:
      - lib/graphql/operations.graphql
    builders:
      artemis:
        options:
          schema_mapping:
            - output: lib/graphql/graphql_api.dart
              schema: lib/graphql/schema.graphql
```
</details>
<details>
<summary>Artemis output:</summary>
  
```bash
# Please paste the output of
$ flutter pub run build_runner build --verbose
[ +313 ms] executing: sysctl hw.optional.arm64
[  +34 ms] Exit code 1 from: sysctl hw.optional.arm64
[   +1 ms] sysctl: unknown oid 'hw.optional.arm64'
[  +13 ms] executing: [/Users/admin/flutter-sdk/flutter/] git -c log.showSignature=false log -n 1
--pretty=format:%H
[  +50 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] e384ca7979097922e08ea30b960759bf6dfa1e60
[   +3 ms] executing: [/Users/admin/flutter-sdk/flutter/] git tag --points-at
e384ca7979097922e08ea30b960759bf6dfa1e60
[+1440 ms] Exit code 0 from: git tag --points-at e384ca7979097922e08ea30b960759bf6dfa1e60
[   +4 ms] executing: [/Users/admin/flutter-sdk/flutter/] git describe --match *.*.* --long --tags
e384ca7979097922e08ea30b960759bf6dfa1e60
[ +167 ms] Exit code 0 from: git describe --match *.*.* --long --tags
e384ca7979097922e08ea30b960759bf6dfa1e60
[   +1 ms] 2.1.0-12.0.pre-288-ge384ca7979
[  +12 ms] executing: [/Users/admin/flutter-sdk/flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +43 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/master
[        ] executing: [/Users/admin/flutter-sdk/flutter/] git ls-remote --get-url origin
[  +36 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[ +215 ms] executing: [/Users/admin/flutter-sdk/flutter/] git rev-parse --abbrev-ref HEAD
[  +20 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +1 ms] master
[  +20 ms] executing: sw_vers -productName
[  +25 ms] Exit code 0 from: sw_vers -productName
[   +3 ms] macOS
[        ] executing: sw_vers -productVersion
[  +24 ms] Exit code 0 from: sw_vers -productVersion
[        ] 11.3
[   +1 ms] executing: sw_vers -buildVersion
[  +72 ms] Exit code 0 from: sw_vers -buildVersion
[   +2 ms] 20E5217a
[ +142 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +4 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +60 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[        ] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'PubDependencies' is not required, skipping update.
[  +49 ms] Using /Users/admin/flutter-sdk/flutter/.pub-cache for the pub cache.
[        ] executing: /Users/admin/flutter-sdk/flutter/bin/cache/dart-sdk/bin/pub run build_runner
build --verbose
[INFO] Generating build script...
[INFO] Generating build script completed, took 712ms

[WARNING] build.fallback:
The package `vivamosolo_app` does not include some required sources in any of its targets (see their build.yaml file).
The missing sources are:
  - $package$
  - lib/$lib$
[INFO] BuildDefinition:Initializing inputs
[INFO] BuildDefinition:Reading cached asset graph...
[INFO] BuildDefinition:Reading cached asset graph completed, took 153ms

[INFO] BuildDefinition:Checking for updates since last build...
[INFO] BuildDefinition:Checking for updates since last build completed, took 759ms

[INFO] Build:Running build...
[INFO] Build:Running build completed, took 12ms

[INFO] Build:Caching finalized dependency graph...
[INFO] Build:Caching finalized dependency graph completed, took 135ms

[INFO] Build:Succeeded after 172ms with 0 outputs (0 actions)

[+6064 ms] "flutter run" took 6,230ms.
[  +80 ms] ensureAnalyticsSent: 73ms
[   +2 ms] Running shutdown hooks
[   +1 ms] Shutdown hooks complete
[   +3 ms] exiting with code 0
```
</details>
<details>
<summary>GraphQL schema:</summary>

```graphql
type Period {
  id: ID!
  createdAt: DateTime!
  updatedAt: DateTime!
  active: Boolean!
  index: Int!
  progression: Float!
  completeAt: DateTime!
  groupId: String!
  group: Group!
  payments: [Payment!]
}

# A date-time string at UTC, such as 2019-12-03T09:54:33Z, compliant with the date-time format.
scalar DateTime

type Payment {
  id: ID!
  createdAt: DateTime!
  updatedAt: DateTime!
  amount: Float!
  status: PaymentStatus!
  groupId: String!
  memberId: String!
  periodId: String!
  group: Group!
  member: Member!
  period: Period!
}

enum PaymentStatus {
  PENDING
  COMPLETED
  FAILED
  CANCELLED
}

type Member {
  id: ID!
  createdAt: DateTime!
  updatedAt: DateTime!
  groupId: String!
  userId: String!
  group: Group!
  user: User!
  payments: [Payment!]
}

type User {
  id: ID!
  email: String!
  name: String!
  surname: String!
  referenceId: String!
  createdAt: DateTime!
  updatedAt: DateTime!
  adminGroups: [Group!]
  asMembers: [Member!]
}

type Group {
  id: ID!
  createdAt: DateTime!
  updatedAt: DateTime!
  name: String!
  code: String!
  balance: Float!
  payment: Float!
  cycle: GroupCycle!
  adminId: String!
  admin: User!
  periods: [Period!]
  members: [Member!]
  payments: [Payment!]
}

enum GroupCycle {
  DAILY
  WEEKLY
  BIWEEKLY
  MONTHLY
}

type Query {
  findGroup(where: GroupWhereUniqueInput!): Group!
  listGroups(
    where: GroupWhereInput
    orderBy: [GroupOrderByInput!]
    cursor: GroupWhereUniqueInput
    take: Int
    skip: Int
    distinct: [GroupScalarFieldEnum!]
  ): [Group!]!
  findUser(where: UserWhereUniqueInput!): User!
  listUsers(
    where: UserWhereInput
    orderBy: [UserOrderByInput!]
    cursor: UserWhereUniqueInput
    take: Int
    skip: Int
    distinct: [UserScalarFieldEnum!]
  ): [User!]!
}

input GroupWhereUniqueInput {
  id: String
  adminId_name: GroupAdminIdNameCompoundUniqueInput
}

input GroupAdminIdNameCompoundUniqueInput {
  adminId: String!
  name: String!
}

input GroupWhereInput {
  AND: [GroupWhereInput!]
  OR: [GroupWhereInput!]
  NOT: [GroupWhereInput!]
  id: StringFilter
  createdAt: DateTimeFilter
  updatedAt: DateTimeFilter
  name: StringFilter
  code: StringFilter
  balance: FloatFilter
  payment: FloatFilter
  cycle: EnumGroupCycleFilter
  adminId: StringFilter
  admin: UserRelationFilter
  periods: PeriodListRelationFilter
  members: MemberListRelationFilter
  payments: PaymentListRelationFilter
}

input StringFilter {
  equals: String
  in: [String!]
  notIn: [String!]
  lt: String
  lte: String
  gt: String
  gte: String
  contains: String
  startsWith: String
  endsWith: String
  mode: QueryMode
  not: NestedStringFilter
}

enum QueryMode {
  default
  insensitive
}

input NestedStringFilter {
  equals: String
  in: [String!]
  notIn: [String!]
  lt: String
  lte: String
  gt: String
  gte: String
  contains: String
  startsWith: String
  endsWith: String
  not: NestedStringFilter
}

input DateTimeFilter {
  equals: DateTime
  in: [DateTime!]
  notIn: [DateTime!]
  lt: DateTime
  lte: DateTime
  gt: DateTime
  gte: DateTime
  not: NestedDateTimeFilter
}

input NestedDateTimeFilter {
  equals: DateTime
  in: [DateTime!]
  notIn: [DateTime!]
  lt: DateTime
  lte: DateTime
  gt: DateTime
  gte: DateTime
  not: NestedDateTimeFilter
}

input FloatFilter {
  equals: Float
  in: [Float!]
  notIn: [Float!]
  lt: Float
  lte: Float
  gt: Float
  gte: Float
  not: NestedFloatFilter
}

input NestedFloatFilter {
  equals: Float
  in: [Float!]
  notIn: [Float!]
  lt: Float
  lte: Float
  gt: Float
  gte: Float
  not: NestedFloatFilter
}

input EnumGroupCycleFilter {
  equals: GroupCycle
  in: [GroupCycle!]
  notIn: [GroupCycle!]
  not: NestedEnumGroupCycleFilter
}

input NestedEnumGroupCycleFilter {
  equals: GroupCycle
  in: [GroupCycle!]
  notIn: [GroupCycle!]
  not: NestedEnumGroupCycleFilter
}

input UserRelationFilter {
  is: UserWhereInput
  isNot: UserWhereInput
}

input UserWhereInput {
  AND: [UserWhereInput!]
  OR: [UserWhereInput!]
  NOT: [UserWhereInput!]
  id: StringFilter
  email: StringFilter
  name: StringFilter
  surname: StringFilter
  referenceId: StringFilter
  createdAt: DateTimeFilter
  updatedAt: DateTimeFilter
  adminGroups: GroupListRelationFilter
  asMembers: MemberListRelationFilter
}

input GroupListRelationFilter {
  every: GroupWhereInput
  some: GroupWhereInput
  none: GroupWhereInput
}

input MemberListRelationFilter {
  every: MemberWhereInput
  some: MemberWhereInput
  none: MemberWhereInput
}

input MemberWhereInput {
  AND: [MemberWhereInput!]
  OR: [MemberWhereInput!]
  NOT: [MemberWhereInput!]
  id: StringFilter
  createdAt: DateTimeFilter
  updatedAt: DateTimeFilter
  groupId: StringFilter
  userId: StringFilter
  group: GroupRelationFilter
  user: UserRelationFilter
  payments: PaymentListRelationFilter
}

input GroupRelationFilter {
  is: GroupWhereInput
  isNot: GroupWhereInput
}

input PaymentListRelationFilter {
  every: PaymentWhereInput
  some: PaymentWhereInput
  none: PaymentWhereInput
}

input PaymentWhereInput {
  AND: [PaymentWhereInput!]
  OR: [PaymentWhereInput!]
  NOT: [PaymentWhereInput!]
  id: StringFilter
  createdAt: DateTimeFilter
  updatedAt: DateTimeFilter
  amount: FloatFilter
  status: EnumPaymentStatusFilter
  groupId: StringFilter
  memberId: StringFilter
  periodId: StringFilter
  group: GroupRelationFilter
  member: MemberRelationFilter
  period: PeriodRelationFilter
}

input EnumPaymentStatusFilter {
  equals: PaymentStatus
  in: [PaymentStatus!]
  notIn: [PaymentStatus!]
  not: NestedEnumPaymentStatusFilter
}

input NestedEnumPaymentStatusFilter {
  equals: PaymentStatus
  in: [PaymentStatus!]
  notIn: [PaymentStatus!]
  not: NestedEnumPaymentStatusFilter
}

input MemberRelationFilter {
  is: MemberWhereInput
  isNot: MemberWhereInput
}

input PeriodRelationFilter {
  is: PeriodWhereInput
  isNot: PeriodWhereInput
}

input PeriodWhereInput {
  AND: [PeriodWhereInput!]
  OR: [PeriodWhereInput!]
  NOT: [PeriodWhereInput!]
  id: StringFilter
  createdAt: DateTimeFilter
  updatedAt: DateTimeFilter
  active: BoolFilter
  index: IntFilter
  progression: FloatFilter
  completeAt: DateTimeFilter
  groupId: StringFilter
  group: GroupRelationFilter
  payments: PaymentListRelationFilter
}

input BoolFilter {
  equals: Boolean
  not: NestedBoolFilter
}

input NestedBoolFilter {
  equals: Boolean
  not: NestedBoolFilter
}

input IntFilter {
  equals: Int
  in: [Int!]
  notIn: [Int!]
  lt: Int
  lte: Int
  gt: Int
  gte: Int
  not: NestedIntFilter
}

input NestedIntFilter {
  equals: Int
  in: [Int!]
  notIn: [Int!]
  lt: Int
  lte: Int
  gt: Int
  gte: Int
  not: NestedIntFilter
}

input PeriodListRelationFilter {
  every: PeriodWhereInput
  some: PeriodWhereInput
  none: PeriodWhereInput
}

input GroupOrderByInput {
  id: SortOrder
  createdAt: SortOrder
  updatedAt: SortOrder
  name: SortOrder
  code: SortOrder
  balance: SortOrder
  payment: SortOrder
  cycle: SortOrder
  adminId: SortOrder
}

enum SortOrder {
  asc
  desc
}

enum GroupScalarFieldEnum {
  id
  createdAt
  updatedAt
  name
  code
  balance
  payment
  cycle
  adminId
}

input UserWhereUniqueInput {
  id: String
  email: String
  referenceId: String
}

input UserOrderByInput {
  id: SortOrder
  email: SortOrder
  name: SortOrder
  surname: SortOrder
  referenceId: SortOrder
  createdAt: SortOrder
  updatedAt: SortOrder
}

enum UserScalarFieldEnum {
  id
  email
  name
  surname
  referenceId
  createdAt
  updatedAt
}

type Mutation {
  createGroup(data: GroupCreateInput!): Group!
  createUser(data: UserCreateInput!): User!
}

input GroupCreateInput {
  id: String
  createdAt: DateTime
  updatedAt: DateTime
  name: String!
  code: String!
  balance: Float!
  payment: Float!
  cycle: GroupCycle!
  admin: UserCreateNestedOneWithoutAdminGroupsInput!
  periods: PeriodCreateNestedManyWithoutGroupInput
  members: MemberCreateNestedManyWithoutGroupInput
  payments: PaymentCreateNestedManyWithoutGroupInput
}

input UserCreateNestedOneWithoutAdminGroupsInput {
  create: UserCreateWithoutAdminGroupsInput
  connectOrCreate: UserCreateOrConnectWithoutAdminGroupsInput
  connect: UserWhereUniqueInput
}

input UserCreateWithoutAdminGroupsInput {
  id: String
  email: String!
  name: String!
  surname: String!
  referenceId: String!
  createdAt: DateTime
  updatedAt: DateTime
  asMembers: MemberCreateNestedManyWithoutUserInput
}

input MemberCreateNestedManyWithoutUserInput {
  create: [MemberCreateWithoutUserInput!]
  connectOrCreate: [MemberCreateOrConnectWithoutUserInput!]
  connect: [MemberWhereUniqueInput!]
}

input MemberCreateWithoutUserInput {
  id: String
  createdAt: DateTime
  updatedAt: DateTime
  group: GroupCreateNestedOneWithoutMembersInput!
  payments: PaymentCreateNestedManyWithoutMemberInput
}

input GroupCreateNestedOneWithoutMembersInput {
  create: GroupCreateWithoutMembersInput
  connectOrCreate: GroupCreateOrConnectWithoutMembersInput
  connect: GroupWhereUniqueInput
}

input GroupCreateWithoutMembersInput {
  id: String
  createdAt: DateTime
  updatedAt: DateTime
  name: String!
  code: String!
  balance: Float!
  payment: Float!
  cycle: GroupCycle!
  admin: UserCreateNestedOneWithoutAdminGroupsInput!
  periods: PeriodCreateNestedManyWithoutGroupInput
  payments: PaymentCreateNestedManyWithoutGroupInput
}

input PeriodCreateNestedManyWithoutGroupInput {
  create: [PeriodCreateWithoutGroupInput!]
  connectOrCreate: [PeriodCreateOrConnectWithoutGroupInput!]
  connect: [PeriodWhereUniqueInput!]
}

input PeriodCreateWithoutGroupInput {
  id: String
  createdAt: DateTime
  updatedAt: DateTime
  active: Boolean!
  index: Int!
  progression: Float
  completeAt: DateTime!
  payments: PaymentCreateNestedManyWithoutPeriodInput
}

input PaymentCreateNestedManyWithoutPeriodInput {
  create: [PaymentCreateWithoutPeriodInput!]
  connectOrCreate: [PaymentCreateOrConnectWithoutPeriodInput!]
  connect: [PaymentWhereUniqueInput!]
}

input PaymentCreateWithoutPeriodInput {
  id: String
  createdAt: DateTime
  updatedAt: DateTime
  amount: Float!
  status: PaymentStatus!
  group: GroupCreateNestedOneWithoutPaymentsInput!
  member: MemberCreateNestedOneWithoutPaymentsInput!
}

input GroupCreateNestedOneWithoutPaymentsInput {
  create: GroupCreateWithoutPaymentsInput
  connectOrCreate: GroupCreateOrConnectWithoutPaymentsInput
  connect: GroupWhereUniqueInput
}

input GroupCreateWithoutPaymentsInput {
  id: String
  createdAt: DateTime
  updatedAt: DateTime
  name: String!
  code: String!
  balance: Float!
  payment: Float!
  cycle: GroupCycle!
  admin: UserCreateNestedOneWithoutAdminGroupsInput!
  periods: PeriodCreateNestedManyWithoutGroupInput
  members: MemberCreateNestedManyWithoutGroupInput
}

input MemberCreateNestedManyWithoutGroupInput {
  create: [MemberCreateWithoutGroupInput!]
  connectOrCreate: [MemberCreateOrConnectWithoutGroupInput!]
  connect: [MemberWhereUniqueInput!]
}

input MemberCreateWithoutGroupInput {
  id: String
  createdAt: DateTime
  updatedAt: DateTime
  user: UserCreateNestedOneWithoutAsMembersInput!
  payments: PaymentCreateNestedManyWithoutMemberInput
}

input UserCreateNestedOneWithoutAsMembersInput {
  create: UserCreateWithoutAsMembersInput
  connectOrCreate: UserCreateOrConnectWithoutAsMembersInput
  connect: UserWhereUniqueInput
}

input UserCreateWithoutAsMembersInput {
  id: String
  email: String!
  name: String!
  surname: String!
  referenceId: String!
  createdAt: DateTime
  updatedAt: DateTime
  adminGroups: GroupCreateNestedManyWithoutAdminInput
}

input GroupCreateNestedManyWithoutAdminInput {
  create: [GroupCreateWithoutAdminInput!]
  connectOrCreate: [GroupCreateOrConnectWithoutAdminInput!]
  connect: [GroupWhereUniqueInput!]
}

input GroupCreateWithoutAdminInput {
  id: String
  createdAt: DateTime
  updatedAt: DateTime
  name: String!
  code: String!
  balance: Float!
  payment: Float!
  cycle: GroupCycle!
  periods: PeriodCreateNestedManyWithoutGroupInput
  members: MemberCreateNestedManyWithoutGroupInput
  payments: PaymentCreateNestedManyWithoutGroupInput
}

input PaymentCreateNestedManyWithoutGroupInput {
  create: [PaymentCreateWithoutGroupInput!]
  connectOrCreate: [PaymentCreateOrConnectWithoutGroupInput!]
  connect: [PaymentWhereUniqueInput!]
}

input PaymentCreateWithoutGroupInput {
  id: String
  createdAt: DateTime
  updatedAt: DateTime
  amount: Float!
  status: PaymentStatus!
  member: MemberCreateNestedOneWithoutPaymentsInput!
  period: PeriodCreateNestedOneWithoutPaymentsInput!
}

input MemberCreateNestedOneWithoutPaymentsInput {
  create: MemberCreateWithoutPaymentsInput
  connectOrCreate: MemberCreateOrConnectWithoutPaymentsInput
  connect: MemberWhereUniqueInput
}

input MemberCreateWithoutPaymentsInput {
  id: String
  createdAt: DateTime
  updatedAt: DateTime
  group: GroupCreateNestedOneWithoutMembersInput!
  user: UserCreateNestedOneWithoutAsMembersInput!
}

input MemberCreateOrConnectWithoutPaymentsInput {
  where: MemberWhereUniqueInput!
  create: MemberCreateWithoutPaymentsInput!
}

input MemberWhereUniqueInput {
  id: String
  groupId_userId: MemberGroupIdUserIdCompoundUniqueInput
}

input MemberGroupIdUserIdCompoundUniqueInput {
  groupId: String!
  userId: String!
}

input PeriodCreateNestedOneWithoutPaymentsInput {
  create: PeriodCreateWithoutPaymentsInput
  connectOrCreate: PeriodCreateOrConnectWithoutPaymentsInput
  connect: PeriodWhereUniqueInput
}

input PeriodCreateWithoutPaymentsInput {
  id: String
  createdAt: DateTime
  updatedAt: DateTime
  active: Boolean!
  index: Int!
  progression: Float
  completeAt: DateTime!
  group: GroupCreateNestedOneWithoutPeriodsInput!
}

input GroupCreateNestedOneWithoutPeriodsInput {
  create: GroupCreateWithoutPeriodsInput
  connectOrCreate: GroupCreateOrConnectWithoutPeriodsInput
  connect: GroupWhereUniqueInput
}

input GroupCreateWithoutPeriodsInput {
  id: String
  createdAt: DateTime
  updatedAt: DateTime
  name: String!
  code: String!
  balance: Float!
  payment: Float!
  cycle: GroupCycle!
  admin: UserCreateNestedOneWithoutAdminGroupsInput!
  members: MemberCreateNestedManyWithoutGroupInput
  payments: PaymentCreateNestedManyWithoutGroupInput
}

input GroupCreateOrConnectWithoutPeriodsInput {
  where: GroupWhereUniqueInput!
  create: GroupCreateWithoutPeriodsInput!
}

input PeriodCreateOrConnectWithoutPaymentsInput {
  where: PeriodWhereUniqueInput!
  create: PeriodCreateWithoutPaymentsInput!
}

input PeriodWhereUniqueInput {
  id: String
}

input PaymentCreateOrConnectWithoutGroupInput {
  where: PaymentWhereUniqueInput!
  create: PaymentCreateWithoutGroupInput!
}

input PaymentWhereUniqueInput {
  id: String
}

input GroupCreateOrConnectWithoutAdminInput {
  where: GroupWhereUniqueInput!
  create: GroupCreateWithoutAdminInput!
}

input UserCreateOrConnectWithoutAsMembersInput {
  where: UserWhereUniqueInput!
  create: UserCreateWithoutAsMembersInput!
}

input PaymentCreateNestedManyWithoutMemberInput {
  create: [PaymentCreateWithoutMemberInput!]
  connectOrCreate: [PaymentCreateOrConnectWithoutMemberInput!]
  connect: [PaymentWhereUniqueInput!]
}

input PaymentCreateWithoutMemberInput {
  id: String
  createdAt: DateTime
  updatedAt: DateTime
  amount: Float!
  status: PaymentStatus!
  group: GroupCreateNestedOneWithoutPaymentsInput!
  period: PeriodCreateNestedOneWithoutPaymentsInput!
}

input PaymentCreateOrConnectWithoutMemberInput {
  where: PaymentWhereUniqueInput!
  create: PaymentCreateWithoutMemberInput!
}

input MemberCreateOrConnectWithoutGroupInput {
  where: MemberWhereUniqueInput!
  create: MemberCreateWithoutGroupInput!
}

input GroupCreateOrConnectWithoutPaymentsInput {
  where: GroupWhereUniqueInput!
  create: GroupCreateWithoutPaymentsInput!
}

input PaymentCreateOrConnectWithoutPeriodInput {
  where: PaymentWhereUniqueInput!
  create: PaymentCreateWithoutPeriodInput!
}

input PeriodCreateOrConnectWithoutGroupInput {
  where: PeriodWhereUniqueInput!
  create: PeriodCreateWithoutGroupInput!
}

input GroupCreateOrConnectWithoutMembersInput {
  where: GroupWhereUniqueInput!
  create: GroupCreateWithoutMembersInput!
}

input MemberCreateOrConnectWithoutUserInput {
  where: MemberWhereUniqueInput!
  create: MemberCreateWithoutUserInput!
}

input UserCreateOrConnectWithoutAdminGroupsInput {
  where: UserWhereUniqueInput!
  create: UserCreateWithoutAdminGroupsInput!
}

input UserCreateInput {
  id: String
  email: String!
  name: String!
  surname: String!
  referenceId: String!
  createdAt: DateTime
  updatedAt: DateTime
  adminGroups: GroupCreateNestedManyWithoutAdminInput
  asMembers: MemberCreateNestedManyWithoutUserInput
}
```
</details>
<details>
<summary>GraphQL query:</summary>

```graphql
fragment UserFragment on User {
    id
    email
    name
    surname
    referenceId
    createdAt
}

mutation createUser($data: UserCreateInput!) {
  createUser(data: $data) {
    ...UserFragment
  }
}

query findUser($email: String!) {
    findUser(where: { email: $email }) {
      ...UserFragment
    }
}

query listGroups($userId: String!) {
    listGroups(where: {
      members: {
        some: {
          userId: {
            equals: $userId
          }
        }
      }
    }) {
      id
      name
      code
      balance
      payment
      cycle
      adminId
    }
}
```
</details>

**Comment by vasilich6107** [comment]
@seyaobey try to add `- lib/**` to your `sources:` section of `build.yaml`

**Comment by seyaobey** [comment]
@vasilich6107 thanks for the suggestion. I added `- lib/**` to my `build.yaml`. But I'm still getting the error:
`pub finished with exit code 1`.

Here is my change:
![image](https://user-images.githubusercontent.com/2738569/113619295-185dfa80-9659-11eb-881f-30bcab6a29a1.png)


And here is my output:
![image](https://user-images.githubusercontent.com/2738569/113619401-36c3f600-9659-11eb-95b4-e5aac36b7348.png)

**Comment by vasilich6107** [comment]
Ok. Let’s do next.
Add to sources only folder with graphql related files

`- lib/graphql/**`

**Comment by seyaobey** [comment]
@vasilich6107 So I cleaned up the `lib/graph/` folder which contained other non-graphql files. I just kept my `schema.graphql` and my `operations.graphql` (which contains my queries). Now running the command, I don't get an error
but:
`[INFO] Succeeded after 79ms with 0 outputs (0 actions)`
And no file produced. I would like to mention that this is the way I indicate my output file (in case it might be wrong):
```
builders:
      artemis:
        options:
          schema_mapping:
            - output: lib/graphql/graphql_api.dart
```

**Comment by seyaobey** [comment]
Also not sure if this can help you:
in my `operations.graphql`, I deliberately introduced an error to see what the output could be. So in my query, I added a field which does not exist in the schema.
But the command existed normally (without output and without any syntax error...)

**Comment by vasilich6107** [comment]
@seyaobey 
here is what you need
https://tppr.me/e4sYX

**Comment by seyaobey** [comment]
@vasilich6107 It worked! All files have been generated as expected. Thank you very much! 
The only problem is that the code generated clashes with `null-safety` mode. (This is probably beyond the scope of this ticket). For this reason, I'll disable Artemis for now. Any plan on supporting `null-safety` in a near future?

**Comment by vasilich6107** [comment]
Null safety is available in 7.0.0-beta.1

**Comment by comigor** [comment]
@seyaobey We've just merged https://github.com/comigor/artemis/pull/283 with NNBD support for Artemis.
I'm closing this but feel free to open another issue, if applicable, after testing the new version `7.0.0-beta.2`.
